base.mk

```diff
@@ -122,7 +122,7 @@ CFLAGS += -DITA_NO_ASSERT
 $(info Asserts will be switched off in debug mode due to code size limitation)
 endif
 else ifeq ($(BUILD), release)
-CFLAGS += -Os -fomit-frame-pointer
+CFLAGS += -Os -fomit-frame-pointer -flto
```

Comment:
@keton can you please update this PR? We released a new version of QMSI last week, so a rebase to start with would be great. I still want to merge the rtc_read() and want you to check if the way we enabled LTO for now is fine or not, please.

I don't think we are going forward with the commit changing our debug builds from -O0 to -Og, however, and I hope that is not a big blocker for you. Let me know if otherwise, ok?

thanks!